main.js

```diff
@@ -444,14 +450,15 @@ function userInput(cmd, DiscordOrigin, discordMsg) {
 
 function stopMsg(discordOrigin, discordMsg, stoppedThing) {
 	msg(discordOrigin, discordMsg, stoppedThing, stoppedThing + " is **stopped**");
+	dc.user.setActivity(stoppedThing + " is stopped.");
```

Comment:
you should use my `activity()` function instead because it checks if the discord bot in enabled

Reply:
Got it, change made and seems to work right. Thanks!

Reply:
When I started working on this project it was the first time I used JS. It nice see you and many others contributing lately